### ISSUE TITLE
Focus returns to the same window after (trying to) switch to another app/window

### ISSUE THREAD

**Issue by methadata** [paperwm/PaperWM#517]
**Describe the bug**
When switching to other windows (Super+arrows), the focus goes back always to the same window.

The only way I know to recover from it, is to logout, or turn the extension off. During the same session, if I turn the extension on again, the bug is still present, so it makes the environment unusable.

This started after upgrading to Fedora 38 / GNOME 44

**To Reproduce**
This happens sporadically. I don't know what triggers it.

**Expected behavior**
Focus shouldn't return to the previous window.

**Screenshots**
n/a

**System information:**
```
Distribution: Fedora Linux
GNOME Shell 44.0
Display server: Wayland
PaperWM branch/tag: fixes-gnome-44-support
PaperWM commit: efb3cb681a473664955d9543a14a78d3332ac978
Enabled extensions:
- appindicatorsupport@rgcjonas.gmail.com
- switcher@landau.fi
- hue-lights@chlumskyvaclav.gmail.com
- user-theme@gnome-shell-extensions.gcampax.github.com
```

**Comment by jtaala** [comment]
hmm - maybe related to #516?  Could you please try change the keybind for switching windows and try again?

There's a `Super+left/right` clash in gnome 44 with the default `Super+left/right` binds in default gnome.

Alternatively, you can disable the default gnome `super+left/right` keybinds (which tile windows to the left or right).  E.g. open `settings` --> select `Keyboard` --> select `View and Customize Shortcuts` --> type the word "split" --> will show the default gnome shortcuts which should be `Super+left/right` --> click each one and hit backspace (which will disable those keybinds).

**Comment by methadata** [comment]
To avoid #516 , I already disabled all keybindings in gnome 44 that clashed with PaperWM keybindings, when I upgraded to gnome 44. 

I've just changed the bind to for left/right window switching, and the bug is still present. I will also try it from a new session to be 100% sure.

It seems to me that maybe it could be related with #495 , as I also loose control of some windows using the mouse when this happens, as explained in that bug. Some other windows work just fine, but I didn't find a pattern yet.

**Comment by jtaala** [comment]
Hmmm - something else must be going on.  I'm also (and I believe @crispinb as well, and definitely @fumbles) are on Fedora 38 / gnome 44 and I'm not seeing this, and haven't been able to reproduce.

#495 was caused by an `end_grab_op` deprecated issue that's been resolved.

Looks like we need a log dump - can you open up a terminal and do something like 

```
journalctl -f -n30
```

and then try reproduce (output of interest will usually be in yellow text) and post here.

Also do a `git pull` for good measure.

**Comment by jtaala** [comment]
Also, to help track this down - consider disabling other extensions, at least temporarily for testing and until we can reproduce - just to rule other extensions out.

**Comment by crispinb** [comment]
Just to confirm - yes I'm on Fedora 38 / Gnome 44 & am not experiencing this problem.

**Comment by jtaala** [comment]
Shout-out to any other Gnome 44 users - are you seeing this?  If so, we need a little help in reproducing this and isolating it.

**Comment by fumbles** [comment]
I've seen the focus go back to another window maybe twice so far. It was before I pulled today though.

**Comment by jtaala** [comment]
Thanks @crispinb & @fumbles! - please keep an eye for this one while using/testing and if you find anything (e.g. certain sequence of events that led up to it, or anything in the logs etc.) please let me know.

Cheers.

**Comment by methadata** [comment]
Yesterday, after last pull (commit 9918ed4), the bug hasn't reproduced in the last 24 hours.

@jtaala If it's ok to you, I could keep like this another 24 hours, and if it still don't show up, I will restore keybindings.

**Comment by methadata** [comment]
Keybindings restored and no issues in one week. I'd recommend to close this issue, as it was solved in 9918ed4.

Thanks!
